reliquery_ui/routers/relics.py

```diff
@@ -112,6 +112,26 @@ async def reliquery_json(
         )
         return json.dumps(relic.get_json(json_name), sort_keys=True, indent=4)
 
+    @router.get(
+        "/reliquery/{storage_name}/{relic_type}/{name}/pandasdf/{pandasdf}",
+        response_class=HTMLResponse,
+    )
+    async def reliquery_pandas_df(
+        storage_name: str, relic_type: str, name: str, pandasdf: str
+    ) -> str:
+        if not Relic.relic_exists(
+            name=name, relic_type=relic_type, storage_name=storage_name
+        ):
+            raise HTTPException(status_code=404, detail="Relic not found")
+
+        relic = Relic(
+            name=name,
+            relic_type=relic_type,
+            storage_name=storage_name,
+            check_exists=False,
+        )
+        return relic.get_pandasdf(pandasdf).to_html()
```

Comment:
perhaps in the future we can find a nicer way to display DataFrames, but I think this is great for now.